.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    - name: Build
+      run: |
+        npm install
+        npm run build-dev
```

Comment:
Why `build-dev` instead of `build`?  I'm just thinking ahead a bit to where CI is also used for the rest of the packaging for publication in Chrome Web Store.

Reply:
`npm run build` fails 

```
npm error Missing script: "build"
```

https://github.com/gliptak/tabli/actions/runs/12509276845/job/34898525946?pr=1